lib/generators/graphiti/resource_generator.rb

```diff
@@ -117,7 +128,7 @@ def responders?
     end
 
     def generate_controller
-      to = File.join("app/controllers", class_path, "#{file_name.pluralize}_controller.rb")
+      to = File.join("app/controllers", namespace_controllers? ? controller_namespaces_path : class_path, "#{file_name.pluralize}_controller.rb")
```

Comment:
I think this line is a little long. It might be good to decide the path to use on a previous line.